esque/controller/topic_controller.py

```diff
@@ -120,8 +121,24 @@ def update_from_cluster(self, topic: Topic):
 
         topic.is_only_local = False
 
+        message = self._get_last_message(topic)
+        topic.last_message_timestamp = message.timestamp()[1]
+
         return topic
 
+    def _get_last_message(self, local_topic: Topic) -> Optional[Tuple[str, int]]:
+        high_watermark = local_topic.max_offset()["offset"] - 1
+        consumer = MessageConsumer(
+            PING_GROUP_ID,
```

Comment:
I'm not sure about the usage of this group, if you know a better one please tell me

Reply:
For now that's okay I think. Maybe we'll have some anonymous consumer at some point, since for the ping we don't need a full blown consumer group either but until then we can reuse the same group id.